cmd/crc/cmd/config/config.go

```diff
@@ -78,26 +81,49 @@ func less(lhsKey, rhsKey string) bool {
 
 func configurableFields(config *config.Config) string {
 	var fields []string
-	var keys []string
-
-	for key := range config.AllConfigs() {
-		keys = append(keys, key)
+	var buf bytes.Buffer
+	writer := tabwriter.NewWriter(&buf, 0, 8, 1, ' ', tabwriter.TabIndent)
+	for _, keyAndValueType := range keysAndValueType(config) {
+		fmt.Fprintln(writer, keyAndValueType)
 	}
+	writer.Flush()
+	keys := strings.Split(buf.String(), "\n")
 	sort.Slice(keys, func(i, j int) bool {
 		return less(keys[i], keys[j])
 	})
 	for _, key := range keys {
+		if key == "" {
+			continue
+		}
 		fields = append(fields, " * "+key)
 	}
 	return strings.Join(fields, "\n")
 }
+func keysAndValueType(config *config.Config) []string {
+	var keyAndValueType []string
+	for key, value := range config.AllConfigs() {
+		var valueType string
+		switch value.Value.(type) {
+		case int:
+			valueType = "Integer"
```

Comment:
is this what we show to the user? an integer as better descirbed as a number, string as text and boolean as a yes/no

Reply:
Maybe yes, but I'd keep `Integer` instead of `Number`, as it isn't a rational or a Gaussian integer :). To be fair, it should be a `Natural number` (as in a non-negative int)  because negative integers aren't sensible in this context either.

Reply:
@gbraad yes this is what shown to user if you think that this should be described as number, text and true/false then I am OK, I think string is better than text but let's see what other think.

Reply:
`Integer` is not a great way of describing `Memory`, as what you want to know is that it's an integer value in MiB (this is almost solved by the go-units work though). Similarly, `Boolean` is not really useful to know which values are correct (true/false, 1/0, yes/no ?)